src/libserialize/opaque.rs

```diff
@@ -193,6 +194,7 @@ impl<'a> Decoder<'a> {
         self.position += bytes;
     }
 
+    #[inline]
     pub fn read_raw_bytes(&mut self, s: &mut [u8]) -> Result<(), String> {
```

Comment:
The `#[inline]`s on things like this and `write_signed_leb128` seem definitely good, though :+1: